docs/dev/intermediate/custom-config-syncing.md

```diff
@@ -6,281 +6,32 @@ description: An intermediate overview of how to sync custom configs for your Let
 
 # Custom Config Syncing
```

Comment:
Please do not remove the contents of this file. We want to make sure that a way w/o APIs is provided for those pursuing this. You can, however, leave the section about the config syncing APIs and move that above the rest of the article, with text saying that an easier method is available.